providers/curation/github.js

```diff
@@ -82,17 +82,30 @@ class GitHubCurationService {
     const prBranch = await this._getBranchName(info)
     await serviceGithub.gitdata.createReference({ owner, repo, ref: `refs/heads/${prBranch}`, sha })
     await Promise.all(
-      patch.patches.map(
-        throat(1, component =>
-          this._writePatch(userGithub, serviceGithub, info, patch.description, component, prBranch)
-        )
-      )
+      patch.patches.map(throat(1, component => this._writePatch(serviceGithub, info, component, prBranch)))
     )
+    const { type, details, summary, resolution } = patch.constributionInfo
+    const Type = type.charAt(0).toUpperCase() + type.substr(1)
+    const description = `
+**Type:** ${Type}
+
+**Summary:**
+${summary}
+
+**Details:**
+${details}
+
+**Resolution:**
+${resolution}
+
+**Affected definitions**:
+${this.formatDefinitions(patch.patches)}`
+
     const result = await (userGithub || serviceGithub).pullRequests.create({
```

Comment:
Since you removed `userGithub` from the function params, where this param come from?